Digipost/src/main/java/no/digipost/android/gui/MainContentActivity.java

```diff
@@ -432,8 +432,14 @@ else if (drawerListItems[content].equals(getResources().getString(R.string.drawe
             return true;
 
         }else if (drawerListItems[content].equals(getResources().getString(R.string.drawer_help))) {
-            Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(ApiConstants.URL_HELP));
-            startActivity(browserIntent);
+            new AlertDialog.Builder(this).setMessage(getString(R.string.dialog_prompt_open_help_new_window))
+                .setPositiveButton(getString(R.string.yes), new DialogInterface.OnClickListener() {
```

Comment:
Kult! 👍
Men det blir kanskje letteste å forstå om vi bytter ut positivt svar med navn på selve handlingen? typisk: "Fortsett" eller "Åpne" i stedet for ja, slik blir det litt lettere å lese.